third_party/amd/lib/TritonAMDGPUToLLVM/SchedInstructions.cpp

```diff
@@ -512,18 +492,57 @@ struct TritonAMDGPUInsertInstructionSchedHints
     : public triton::impl::TritonAMDGPUInsertInstructionSchedHintsBase<
           TritonAMDGPUInsertInstructionSchedHints> {
 
+  explicit TritonAMDGPUInsertInstructionSchedHints(StringRef variant) {
+    this->variant = std::move(variant.str());
+  }
+
+  void guardFlashAttentionLikeProblems(triton::FuncOp funcOp) {
```

Comment:
This disregards the developer choice and forcefully set for attention. Why is that?

Reply:
`guardFlashAttentionLikeProblems` has been removed in the new implementation